NewKernel_d/test/NewKernel_d/Epick_d.cpp

```diff
@@ -745,6 +745,11 @@ static_assert(std::is_same<CGAL::Dimension_tag<3>,Ker3::Dimension>::value);
 static_assert(std::is_same<CGAL::Dynamic_dimension_tag,Kerd::Dimension>::value);
 static_assert(std::is_same<CGAL::Dimension_tag<2>,CGAL::Ambient_dimension<Ker2::Point_d>::type>::value);
 static_assert(std::is_same<CGAL::Dimension_tag<3>,CGAL::Ambient_dimension<Ker3::Point_d,Ker3>::type>::value);
+
+static_assert(Ker2::Has_filtered_predicates_tag::value);
+static_assert(Ker3::Has_filtered_predicates_tag::value);
+static_assert(Kerd::Has_filtered_predicates_tag::value);
+
```

Comment:
@mglisse please review

Reply:
ok, we can always modify this if we start testing non-filtered kernels.